chore: fix PK env vars passed `cross` builds

b-zee opened the pull request.

joshuef commented on sn_node/src/node/flow_ctrl/tests/cmd_utils.rs:
```diff
@@ -80,69 +79,87 @@ pub(crate) async fn handle_online_cmd(
+        _peer: Peer,
+        dispatcher: &'a Dispatcher,
+    ) -> crate::node::error::Result<Self> {
+        // let node = dispatcher.node();
```
bit of cleanup here

joshuef reviewed (approved):
bors r+

oetyng commented on sn_node/src/node/flow_ctrl/tests/cmd_utils.rs:
```diff
@@ -80,69 +79,67 @@ pub(crate) async fn handle_online_cmd(
-    let mut cmds = dispatcher.process_cmd(cmd).await?;
+pub(crate) struct ProcessAndInspectCmds<'a> {
+    pending_cmds: VecDeque<Cmd>,
+    cmds_to_inspect: VecDeque<usize>,
```
Is this one really necessary?
Seems like we just push and pop from the collection. 
What do we use this additional storsge of indices for?

oetyng commented on sn_node/src/node/flow_ctrl/tests/cmd_utils.rs:
```diff
@@ -80,69 +79,67 @@ pub(crate) async fn handle_online_cmd(
-    //         peer,
-    //     )
-    //     .await?;
+    pub(crate) async fn next(&mut self) -> crate::node::error::Result<Option<&Cmd>> {
+        match self.cmds_to_inspect.pop_front() {
+            Some(index) => {
+                let cmd = self.pending_cmds.get(index);
+                assert!(cmd.is_some());
+                Ok(cmd)
+            }
+            None => {
+                while let Some(cmd) = self.pending_cmds.pop_front() {
+                    if !matches!(cmd, Cmd::SendMsg { .. }) {
```
The fn and its signature can be simolified to Option<Cmd> by matching on the popped cmd, and skipping the index collection.

bochaco commented:
bors r+

b-zee committed:
chore: fix PK env vars passed to `cross` builds

Also uses `cargo-binstall` to use pre-compiled `cross` and `just`
binaries to speed up the workflow.

b-zee committed:
chore: escape cross variables passing

b-zee committed:
chore: remove unused var
